### ISSUE TITLE
[BUG] Pytube no longer downloading anything

### ISSUE THREAD

**Issue by maltium** [pytube/pytube#1490]
**Describe the bug**
Pytube no longer downloads the video/audio and instead hangs indefinitely.

**To Reproduce**
- All videos seem to be affected. This is a particular example https://www.youtube.com/watch?v=UgR8sFVF6UU that I confirmed that isn't working.

My Python code looks like so:
```
  yt = YouTube(url)
  return yt.streams.order_by('bitrate').first().download(
      output_path='/tmp', max_retries=4
  )
```

When I step through the code it seems that in ```pytube/streams.py``` the following code never returns
```
for chunk in request.stream(
                    self.url,
                    timeout=timeout,
                    max_retries=max_retries
                ):
```

If I copy the value of ```self.url``` and paste in the browser, the video opens fine.

**Expected behavior**
Video should download

**System information**
Please provide the following information:
- Python 3.10.6
- Pytube 12.1.2
- Command used to install pytube: ```python -m pip install git+https://github.com/pytube/pytube```

**Comment by lolzhunter** [comment]
yeah im having the same issue but weirdly only for just audio tracks like this for example https://www.youtube.com/watch?v=hy-yyLSxopk

regular videos seem to work fine (moving pictures and audio), but it just stalls and doesnt seem to download audio, even if you download it as a video it stalls

**Comment by Yanall-Boutros** [comment]
I am also having this bug with some videos, doesn't seem to happen as much for short videos

**Comment by amyrot** [comment]
`order_by('bitrate').first()` returns the audio stream of the smallest bitrate in that case. It doesn't look like a bug.
